src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Provider } from "react-redux";
 
 import { store, history } from "./store/configureStore";
 import "bootstrap/dist/css/bootstrap.css";
-import "codemirror/lib/codemirror.css";
+// import "codemirror/lib/codemirror.css";
```

Comment:
not necessary anymore? Shall we just remove it?

Reply:
Yup, missed that one.